Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -23,6 +26,42 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
         initializer: function () {
             this.on('*:logOut', this._logOut);
             this.after('*:activeNavigationChange', this._navigateToFirstItemRoute);
+            this.get('app').on('*:resetContentNavigationItem', this._resetContentNavigationItem, this);
+        },
+
+        /**
+         * Updates content link in 'Content' navigation zone for current selected content
+         *
+         * @method _resetContentNavigationItem
+         * @protected
+         * @param {Event} event
+         */
+        _resetContentNavigationItem: function () {
+            const contentItem = this.get('platformNavigationItems').find(item => {
```

Comment:
I think method `getNavigationItem` do the same thing.

Reply:
Yes, but this one is faster because I'm looking for a specific item in a specific zone. I don't have to iterate through all the zones to get that one specific item :)

Reply:
I agree it's faster around 0.0002ms :D 
I think we should use dedicated methods for finding items if such method exists.

Reply:
Ok, I can agree to this option.